api/handler/v1beta1/job_spec.go

```diff
@@ -298,14 +299,45 @@ func (sv *JobSpecServiceServer) RefreshJobs(req *pb.RefreshJobsRequest, respStre
 		mu:     new(sync.Mutex),
 	})
 
-	if err := sv.jobSvc.Refresh(respStream.Context(), req.ProjectName, req.NamespaceNames, req.JobNames, observers); err != nil {
+	err := sv.jobSvc.Refresh(respStream.Context(), req.ProjectName, req.NamespaceNames, req.JobNames, observers)
+	if err != nil {
 		return status.Errorf(codes.Internal, "failed to refresh jobs: \n%s", err.Error())
 	}
 
 	sv.l.Info("finished job refresh", "time", time.Since(startTime))
 	return nil
 }
 
+func (sv *JobSpecServiceServer) GetDeployJobsStatus(ctx context.Context, req *pb.GetDeployJobsStatusRequest) (*pb.GetDeployJobsStatusResponse, error) {
+	deployID, err := uuid.FromString(req.DeployId)
```

Comment:
We are adding `"github.com/gofrs/uuid"` as dependency for project. Other places we are using `"github.com/google/uuid"`.

For reading with id you can try the following with google uuid
```go
deployID, err := uuid.FromBytes([]byte(req.DeployId))
```

Reply:
updated this.